auth/src/main/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivity.java

```diff
@@ -114,7 +115,8 @@ private void populateIdpList(List<IdpConfig> providers) {
                     mProviders.add(new EmailProvider(this, getFlowParams()));
                     break;
                 case AuthUI.PHONE_VERIFICATION_PROVIDER:
-                    mProviders.add(new PhoneProvider(this, getFlowParams()));
+                    PhoneIdpConfig phoneIdpConfig = (PhoneIdpConfig) idpConfig;
```

Comment:
This class `PhoneIdpConfig` is used here but I don't see it defined in this PR anymore.  I think changes to this file need to be revisited.

Reply:
Oops, gone now.